src/west/app/project.py

```diff
@@ -201,6 +206,12 @@ def do_add_parser(self, parser_adder):
                             MANIFEST_URL; .west is created next to "directory"
                             in this case, and manifest.path points at
                             "directory"''')
+        parser.add_argument('--rename-delay-hack', type=int,
```

Comment:
While I agree this is a hack, does the parameter name need to be so... screamy?
```suggestion
        parser.add_argument('--rename-delay', type=int,
```